src/nomad_tajine_plugin/schema_packages/schema_package.py

```diff
@@ -105,10 +107,28 @@ class Ingredient(EntityReference):
 
     def normalize(self, archive, logger: 'BoundLogger'):
         if not self.lab_id:
-            self.lab_id = self.name
+            self.lab_id = self.name.lower().replace(' ', '_')
 
         super().normalize(archive, logger)
 
+        if not self.reference:
+            logger.debug('IngredientType entry not found. Creating a new one.')
+            try:
+                ingredient_type = IngredientType(
+                    name=self.name,
+                    lab_id=self.lab_id,
+                )
+                self.reference = create_archive(
+                    ingredient_type,
+                    archive,
+                    f'{self.lab_id}.archive.json',
```

Comment:
Would NOMAD warn if that archive is created again? Given that overwrite is set to False?